gridcomps/ExtData2G/ExtDataGridCompNG.F90

```diff
@@ -81,7 +81,8 @@ MODULE MAPL_ExtDataGridComp2G
   integer, parameter         :: MAPL_ExtDataResult        = 3
   character(len=*), parameter :: mol_per_mol = 'mol mol-1'
   character(len=*), parameter :: kg_per_kg = 'kg kg-1'
-  character(len=*), parameter :: emission_units = 'kg m-2 s-1'
+  character(len=*), parameter :: emission_units = 'kg m-2'
+  character(len=*), parameter :: per_second = 's-1'
```

Comment:
parameters really should be ALL_CAPS by MAPL coding standards.  But let's not slow down this PR to fix.    But next time ...